src/MagnumPlugins/AssimpImporter/Test/AssimpImporterTest.cpp

```diff
@@ -131,6 +151,27 @@ constexpr struct {
     {"verbose", ImporterFlag::Verbose}
 };
 
+constexpr struct {
+    const char* name;
+    const char* suffix;
+} ExportedAnimationFileData[]{
+    {"Collada", ".dae"},
+    {"Collada curves", "-curves.dae"},
+    {"Collada curves with handles", "-curves-handles.dae"},
+    {"FBX", ".fbx"},
+    {"glTF", ".gltf"}
+};
+
+constexpr struct {
+    const char* name;
+    const char* suffix;
+} GltfAnimationFileData[]{
+    {"ascii external", ".gltf"},
+    {"ascii embedded", "-embedded.gltf"},
+    {"binary external", ".glb"},
+    {"binary embedded", "-embedded.glb"}
+};
```

Comment:
Does Assimp handle all these correctly the same? If it does, then I would use just a single variant, the one that's the easiest to produce, which is I suppose "ASCII external"?

In case of TinyGltfImporter these were important because unfortunately tinygltf used to have bugs and failed to import certain variants, but for Assimp I don't think we need to care that much :)

Reply:
No difference I can see, I removed everything except for `animation.gltf`.